src/app/containers/ArticleTimestamp/index.stories.jsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import timemachine from 'timemachine';
 import ArticleTimestamp from '.';
 import { timestampGenerator } from './testHelpers';
 import { ServiceContextProvider } from '../../contexts/ServiceContext';
 import services from '../../lib/config/services';
 
+timemachine.config({
+  dateString: 'Tue, 01 Jan 2019 00:00:00 GMT',
```

Comment:
Does it need both of these things? surely just 1 is enough? :)

Reply:
default params for timemachine:

Parameter | Description | Default
-- | -- | --
dateString | Date you want set as a string | Thu, 01 Jan 1970 00:00:00 GMT
timestamp | Date you want set as a timestamp | 0
difference | Time offset in milliseconds | 0
tick | Whether the system clock should continue ticking | false
keepTime | Whether the system time should be kept | false

so we use the timestamp value in our codebase, but if i don't configure the dateString, our storybook thinks we are in the past. So the safest thing to do, i thought is just to set both values (they are the same time and date btw).